slackbridge/factories.py

```diff
@@ -57,6 +57,7 @@ def buildProtocol(self, addr):
 
     def add_user_bot(self, user_bot):
         IRCBot.users[user_bot.user_id] = user_bot
+        IRCBot.slack_nicks.append(user_bot.slack_name)
```

Comment:
Instead of having a separate list of slack names, couldn't you just iterate through the dictionary of users and see if any of them have a `slack_name` that matches? I think that would be a bit cleaner anyway and prevent there being more places that data needs to be updated.